app/controllers/fat_free_crm/entities_controller.rb

```diff
@@ -139,13 +139,20 @@ def ransack_search
   def get_list_of_records(options = {})
     options[:query]  ||= params[:query]                        if params[:query]
     self.current_page  = options[:page]                        if options[:page]
+    filter_by = params["filter_by"]
+    filter_by_value = params["filter_by_value"]
     query, tags        = parse_query_and_tags(options[:query])
     self.current_query = query
     advanced_search = params[:q].present?
     wants = request.format
-
+    statement = ""
+    if filter_by && filter_by_value
+      statement = filter_by + " = '" + filter_by_value + "'"
```

Comment:
query injection attack so you must validate in code that filter_by one of our choices (status, rating or source)